### PR TITLE
feat: add addNoReferrer in common-utils

### PR DESCRIPTION
**Pull request by HenryQW**
This adds no-referrer attribute to images.
Parameters: 

- $: cheerio selector
- source: source selector, string
- target: target attribute name, typically for lazyload imgs, string
- srcPrefix: prefix for src, string
- removeAttr: attributes to remove, array: ['attrA','attrB']

### CODE REVIEW

**Comment by HenryQW** [comment]
image

**Comment by HenryQW** [comment]
顺带合并了好奇心日报路由

File: docs/social-media.md [hunk 1]
```diff
@@ -532,17 +532,15 @@ pageClass: routes
...
 ## 好奇心日报
 
-### 分类
-
-<Route author="WenhuWee" example="/qdaily/category/5" path="/qdaily/category/:id" :paramsDesc="['分类 id，可在分类 URL 找到']"/>
-
 ### 栏目
 
-<Route author="WenhuWee emdoe" example="/qdaily/column/59" path="/qdaily/column/:id" :paramsDesc="['栏目 id，可在栏目 URL 找到']"/>
+<Route author="WenhuWee emdoe SivaGao HenryQW" example="/qdaily/column/59" path="/qdaily/:type/:id" :paramsDesc="['类型，见下表'，'对应 id，可在 URL 找到']">
```
Comment (DIYgod) [inline docs/social-media.md]: 逗号有误

**Comment by DIYgod** [comment]
Merged, thanks!

**Pull request merged**